docs/notebooks/quickstart.ipynb

```diff
@@ -0,0 +1,801 @@
+    "Exporting is straight forward! You can either export directly to the Platform or to a file. At the time of export, the uploaded evidence will be checked against the governance requirements specified on Platform and let you know what's missing."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bddb93de-6139-4f22-9df0-b97019157541",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# export to API\n",
+    "gov.export()\n",
+    "\n",
+    "# export to file\n",
+    "gov.export(\"assessment_export.json\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d6c71569-6152-42d6-aadc-da60a25eca52",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Other ways to label your evidence"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "21ea5e8a-2cb1-41ec-aafb-a10299774725",
+   "metadata": {},
+   "source": [
+    "On the Credo AI Platform, certain governance requirements may only apply to models that are tagged in a certain way. You can get the requirements tags from governance and apply it by passing a dictionary to the `model_tags` of the `Adapter`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fd269b76-3fb7-4288-8b0a-bf16772b5b6a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gov.get_requirement_tags()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "861cdace-f849-4248-bdd7-cd5a86888902",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "adapter = Adapter(governance = gov, model_name='My Model', model_tags={'model_type': 'regression'})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ee605487-1de4-4fe7-8df2-cb8947a260cd",
+   "metadata": {},
+   "source": [
+    "If you need to label your evidence with more information than just `source` you can do that too."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c0797a51",
+   "metadata": {},
+   "source": [
+    "## Lens in 5 minutes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cce779fd",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "Below is a basic example where our goal is to evaluate the above model. We'll break down this code [below](#Breaking-Down-The-Steps).\n",
+    "\n",
+    "Briefly, the code is doing four things:\n",
+    "\n",
+    "* Wrapping ML artifacts (like models and data) in Lens objects\n",
+    "* Initializing an instance of Lens. Lens is the main object that performs evaluations. Under the hood, it creates a `pipeline` of evaluations that are run.\n",
+    "* Add evaluators to Lens.\n",
+    "* Run Lens"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5b4e6a6a",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# set up model and data artifacts\n",
+    "credo_model = ClassificationModel(name=\"credit_default_classifier\", model_like=model)\n",
+    "credo_data = TabularData(\n",
```

Comment:
All Lens artifacts import are missing

Reply:
Not sure what's going on here... this isn't in the notebook.

Reply:
For some reason this was hidden when I opened jupyter-lab, but I could see these cells in vscode. weird.